docs/how-to/integrate-with-s3.md

```diff
@@ -0,0 +1,181 @@
+# Integrate mquery with s3
+
+One very common question is "how to use Mquery with s3". S3 is a file storage
+API is exposed by many open- and closed-source solutions, like Minio or of course
+AWS. Mquery does not support S3 natively but can work with S3 thanks to its
+very flexible plugin system. Unfortunately, this is not completely transparent,
+and S3 deployment is not easy. In this guide, I'll explain how to integrate
+Mquery with existing s3 deployment.
+
+## Requirements
+
+* Linux system with at least 4GB RAM and enough disk space for your samples
+and index
+* Sudo access
+* Mquery is installed natively (not from docker-compose). We will have to
+install additional dependencies, and it's not trivial with existing images.
+To make things simpler, we assume you installed mquery natively (see
```

Comment:
Isn't this sentence just a rewording of the one above?

Reply:
Right, reworded.